### ISSUE TITLE
Add ability to set language detector Seed

### ISSUE THREAD

**Issue by takenspc** [validator/validator#1513]
**Is your feature request related to a problem? Please describe.**
The language detector produces different results from same input because it uses random number.

https://github.com/validator/language-detection/blob/575f9b99a978a0594dfbebc47663f9dd2db42363/src/com/cybozu/labs/langdetect/Detector.java#L239-L246

Using random number is probably good practice in general though we need reproducible results in some cases.

**Describe the solution you'd like**
If you don’t mind, we would like to add a new system property to set language detector seed.

Example:

```sh
java -Dnu.validator.checker.langDetectionSeed=12345 vnu.jar
```

**Describe alternatives you've considered**
Disabling language detector is possible alternatives to make Nu Html Checker more reproducible. Howerver, the language detector catches some type of errors so that we want to continue using language detector.

**Additional context**
None.
